app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t "global.base_title"
```

Comment:
```suggestion
    base_title = t "global.base_title"
```
như này thôi không đc à em

Reply:
Fixed